src/Modules/Helper.js

```diff
@@ -15,6 +15,7 @@ const findChampions = function(champion_list, champion_ids) {
         let champion = champion_list.find(champ => {
```

Comment:
Let's understand why we need this function to be in utils file. Are we using it in multiple modules? If so, let's solve the underlying problem